src/components/ui/Banner.vue

```diff
@@ -14,30 +14,52 @@
     />
 
     <div class="banner-text">
-      Metamask currently does not support signatures using Ledger/Trezor devices
+      Attention! As per
```

Comment:
Suggestion: For a next version, I would consider using localized string i18n so we can have a better audience multi-language reach.